Mathlib/RepresentationTheory/Homological/GroupCohomology/LowDegree.lean

```diff
@@ -626,46 +776,60 @@ variable {G M : Type*} [Mul G] [CommGroup M] [SMul G M]
 
 /-- A function `f : G → M` satisfies the multiplicative 1-cocycle condition if
 `f(gh) = g • f(h) * f(g)` for all `g, h : G`. -/
-def IsMulOneCocycle (f : G → M) : Prop := ∀ g h : G, f (g * h) = g • f h * f g
+def IsMulCocycle₁ (f : G → M) : Prop := ∀ g h : G, f (g * h) = g • f h * f g
+
+@[deprecated (since := "2025-06-25")] alias IsMulOneCocycle := IsMulCocycle₁
 
 /-- A function `f : G × G → M` satisfies the multiplicative 2-cocycle condition if
 `f(gh, j) * f(g, h) = g • f(h, j) * f(g, hj)` for all `g, h : G`. -/
-def IsMulTwoCocycle (f : G × G → M) : Prop :=
+def IsMulCocycle₂ (f : G × G → M) : Prop :=
   ∀ g h j : G, f (g * h, j) * f (g, h) = g • (f (h, j)) * f (g, h * j)
 
+@[deprecated (since := "2025-06-25")] alias IsMulTwoCocycle := IsMulCocycle₂
+
 end
 
 section
 
 variable {G M : Type*} [Monoid G] [CommGroup M] [MulAction G M]
 
-theorem map_one_of_isMulOneCocycle {f : G → M} (hf : IsMulOneCocycle f) :
+theorem map_one_of_isMulCocycle₁ {f : G → M} (hf : IsMulCocycle₁ f) :
     f 1 = 1 := by
   simpa only [mul_one, one_smul, left_eq_mul] using hf 1 1
 
-theorem map_one_fst_of_isMulTwoCocycle {f : G × G → M} (hf : IsMulTwoCocycle f) (g : G) :
+@[deprecated (since := "2025-06-25")] alias map_one_of_isMulOneCocycle := map_one_of_isMulCocycle₁
+
+theorem map_one_fst_of_isMulCocycle₂ {f : G × G → M} (hf : IsMulCocycle₂ f) (g : G) :
     f (1, g) = f (1, 1) := by
   simpa only [one_smul, one_mul, mul_one, mul_right_inj] using (hf 1 1 g).symm
 
-theorem map_one_snd_of_isMulTwoCocycle {f : G × G → M} (hf : IsMulTwoCocycle f) (g : G) :
+@[deprecated (since := "2025-06-25")]
+alias map_one_fst_of_isMulTwoCocycle := map_one_fst_of_isMulCocycle₂
+
+theorem map_one_snd_of_isMulCocycle₂ {f : G × G → M} (hf : IsMulCocycle₂ f) (g : G) :
     f (g, 1) = g • f (1, 1) := by
   simpa only [mul_one, mul_left_inj] using hf g 1 1
 
+@[deprecated (since := "2025-06-25")]
+alias map_one_snd_of_isMulTwoCocycle := map_one_snd_of_isMulCocycle₂
+
 end
 
 section
 
 variable {G M : Type*} [Group G] [CommGroup M] [MulAction G M]
 
-@[scoped simp] theorem map_inv_of_isMulOneCocycle {f : G → M} (hf : IsMulOneCocycle f) (g : G) :
+@[scoped simp] theorem map_inv_of_isMulCocycle₁ {f : G → M} (hf : IsMulCocycle₁ f) (g : G) :
     g • f g⁻¹ = (f g)⁻¹ := by
-  rw [← mul_eq_one_iff_eq_inv, ← map_one_of_isMulOneCocycle hf, ← mul_inv_cancel g, hf g g⁻¹]
+  rw [← mul_eq_one_iff_eq_inv, ← map_one_of_isMulCocycle₁ hf, ← mul_inv_cancel g, hf g g⁻¹]
+
+@[deprecated (since := "2025-06-25")] alias map_inv_of_isMulOneCocycle := map_inv_of_isMulCocycle₁
 
-theorem smul_map_inv_div_map_inv_of_isMulTwoCocycle
-    {f : G × G → M} (hf : IsMulTwoCocycle f) (g : G) :
+theorem smul_map_inv_div_map_inv_of_isMulCocycle₂
```

Comment:
this needs a deprecation

Reply:
oof, my bad, thank you for pointing all these out; I've added them now.